src/plugins/kibana_react/public/page_template/solution_nav/solution_nav.tsx

```diff
@@ -26,6 +34,21 @@ export type KibanaPageTemplateSolutionNavProps = EuiSideNavProps<{}> & {
    * Solution logo, i.e. "logoObservability"
    */
   icon?: KibanaPageTemplateSolutionNavAvatarProps['iconType'];
+  /**
+   * Control the collapsed state
+   */
+  isOpenOnDesktop?: boolean;
+  onCollapse?: () => void;
+};
+
+const negativeTabIndex = (items: Array<EuiSideNavItemType<{}>>) => {
+  return items.map((item) => {
+    // TODO need EUI support
+    // @ts-ignore
```

Comment:
Could we use a specific ignore option here? e.g. `// @ts-ignore-next-line EUI does not support ....`